apollo-runtime/src/main/java/com/apollographql/apollo/internal/RealApolloStore.kt

```diff
@@ -349,15 +355,19 @@ class RealApolloStore(normalizedCache: NormalizedCache, cacheKeyResolver: CacheK
             cacheKeyResolver(),
             cacheHeaders,
             cacheKeyBuilder)
-        val responseReader = RealResponseReader(operation.variables(), rootRecord,
-            fieldValueResolver, customScalarAdapters, responseNormalizer)
+        val responseReader = RealResponseReader(
+            operation.variables(),
+            rootRecord,
+            fieldValueResolver,
+            customScalarAdapters
+        )
         return try {
-          responseNormalizer.willResolveRootQuery(operation)
           val data = operation.adapter().fromResponse(responseReader, null)
+          val records = operation.normalize(data, customScalarAdapters, networkResponseNormalizer() as ResponseNormalizer<Map<String, Any>?>)
```

Comment:
Since we always use `neworkResponseNormalizer` here, we can remove the parameter and do other cleanups. I didn't do them yet in order to not clash with https://github.com/apollographql/apollo-android/pull/2839